src/ooxml/java/org/apache/poi/xddf/usermodel/chart/XDDFChart.java

```diff
@@ -184,68 +217,70 @@ public XDDFShapeProperties getOrAddShapeProperties() {
         }
         return new XDDFShapeProperties(properties);
     }
-
+    
     public void deleteShapeProperties() {
         if (getCTPlotArea().isSetSpPr()) {
             getCTPlotArea().unsetSpPr();
         }
     }
-
+    
     public XDDFChartLegend getOrAddLegend() {
         return new XDDFChartLegend(chart);
     }
-
+    
     public void deleteLegend() {
         if (chart.isSetLegend()) {
             chart.unsetLegend();
         }
     }
-
+    
     public XDDFManualLayout getOrAddManualLayout() {
         return new XDDFManualLayout(chart.getPlotArea());
     }
-
+    
     public void plot(XDDFChartData data) {
+        XSSFSheet sheet = getSheet();
         for (XDDFChartData.Series series : data.getSeries()) {
             series.plot();
+            fillSheet(sheet, series.getCategoryData(), series.getValuesData());
```

Comment:
I am afraid that would break the charts in spreadsheets... Check in `XSSFChart` to be sure.